### PR TITLE
Helm deploy (chart)

### PR DESCRIPTION
**Pull request by Arslanbekov**
#9 Support helm deploy

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,5 @@
 
 *.test
 *.out
-slack-duty-bot
```
Comment (insidieux) [inline .gitignore]: Why removing compiled binary from ignore?

File: .helm/slack-duty-bot/templates/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        - {{ $keyword | quote}}
+      {{- end }}
+    duties:
+      - [ {{ .Values.dutyUsers.sunday | quote }} ]
+      - [ {{ .Values.dutyUsers.monday | quote }} ]
+      - [ {{ .Values.dutyUsers.tuesday | quote }} ]
+      - [ {{ .Values.dutyUsers.wednesday | quote }} ]
+      - [ {{ .Values.dutyUsers.thursday | quote }} ] 
+      - [ {{ .Values.dutyUsers.friday | quote }} ]
+      - [ {{ .Values.dutyUsers.saturday | quote }} ]
```
Comment (insidieux) [inline .helm/slack-duty-bot/templates/configmap.yaml]: Add empty last line

File: .helm/slack-duty-bot/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        - name: "{{ include "slack-duty-bot.name" . }}-config-volume"
+          mountPath: /etc/slack-duty-bot
+        resources:
+{{ toYaml .Values.resources | indent 12 }}
+      volumes:
+        - name: "{{ include "slack-duty-bot.name" . }}-config-volume"
+          configMap:
+            name: "{{ include "slack-duty-bot.name" . }}-config-map"
+      restartPolicy: Always
+      terminationGracePeriodSeconds: 30
```
Comment (insidieux) [inline .helm/slack-duty-bot/templates/deployment.yaml]: Add empty last line

File: .helm/slack-duty-bot/templates/secret.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "{{ include "slack-duty-bot.name" . }}-secret"
+type: Opaque
+data:
+  token: {{ .Values.configuration.slackToken | b64enc }}
```
Comment (insidieux) [inline .helm/slack-duty-bot/templates/secret.yaml]: Add empty last line

File: .helm/slack-duty-bot/values.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  friday:    user.one
+  saturday:  user.one, user.two
+
+resources: {}
+  # limits:
+  #  cpu: 100m
+  #  memory: 128Mi
+  # requests:
+  #  cpu: 100m
+  #  memory: 128Mi
```
Comment (insidieux) [inline .helm/slack-duty-bot/values.yaml]: Add empty last line

File: README.md [hunk 6]
```diff
@@ -95,19 +107,44 @@ duties:
...
 * Config file
 
-### Deploy to Kubernetes
+## Deploy to Kubernetes
+
+### Deploy with Helm
+
+#### Configuration
+
+The following table lists the configurable parameters of the Drupal chart and their default values.
```
Comment (insidieux) [inline README.md]: Drupal ?

File: README.md [hunk 7]
```diff
@@ -95,19 +107,44 @@ duties:
...
+| `image.repository`                | SDB image registry                         | `iqoption/slack-duty-bot`                                 |
+| `image.tag`                       | SDB Image tag                              | `{VERSION}`                                               |
+| `image.pullPolicy`                | SDB image pull policy                      | `IfNotPresent`                                            |
+| `configuration.slackToken`        | Slack token                                | `nil`                                                     |
+| `configuration.keywords`          | Trigger words                              | array `duty`                                              |
+
+#### Run deploy
+
+```bash
+helm upgrade --install slack-duty-bot-my-app-name .helm/slack-duty-bot/ --set configuration.slackToken=secret-token,configuration.keywords[0]="duty",configuration.keywords[1]="autobot",dutyUsers.monday=denis.arslanbekov,dutyUsers.tuesday=pavel.ageev,dutyUsers.wednesday=konstantin.perminov
```
Comment (insidieux) [inline README.md]: Change duty users names to dummy

File: .gitignore (hunk 1 above)
Comment (Arslanbekov) [reply .gitignore]: Thanks, fixed.

File: .helm/slack-duty-bot/templates/configmap.yaml (hunk 2 above)
Comment (Arslanbekov) [reply .helm/slack-duty-bot/templates/configmap.yaml]: Thanks, fixed.

File: .helm/slack-duty-bot/templates/deployment.yaml (hunk 3 above)
Comment (Arslanbekov) [reply .helm/slack-duty-bot/templates/deployment.yaml]: Thanks, fixed.

File: .helm/slack-duty-bot/templates/secret.yaml (hunk 4 above)
Comment (Arslanbekov) [reply .helm/slack-duty-bot/templates/secret.yaml]: Thanks, fixed.

File: .helm/slack-duty-bot/values.yaml (hunk 5 above)
Comment (Arslanbekov) [reply .helm/slack-duty-bot/values.yaml]: Thanks, fixed.

File: README.md (hunk 6 above)
Comment (Arslanbekov) [reply README.md]: Thanks, fixed.

File: README.md (hunk 7 above)
Comment (Arslanbekov) [reply README.md]: Thanks, fixed.

**Pull request merged**